API.md

```diff
@@ -103,6 +103,34 @@ LoRa.endPacket()
 
 Returns `1` on success, `0` on failure.
 
+## Sending data - non blocking
```

Comment:
I don't think we should have a totally new section for non-blocking, rather modify the existing sending section, since there is really just a new parameter to `endPacket` and `beginPacket` has another reason to fail.

@morganrallen any thoughts on this?